README.md

```diff
@@ -17,30 +17,83 @@ Skycoin is small part of OP Redecentralize and OP Darknet Plan.
 * [Skycoin Blockchain Explorer](https://explorer.skycoin.net)
 * [Skycoin Distribution Event](https://event.skycoin.net)
 
+## Table of Contents
+
+<!-- MarkdownTOC depth="2" autolink="true" bracket="round" -->
+
+- [Installation](#installation)
+    - [Installing Go](#installing-go)
+    - [Set GOPATH in your environment](#set-gopath-in-your-environment)
+    - [Go get skycoin](#go-get-skycoin)
+    - [Run Skycoin from the command line](#run-skycoin-from-the-command-line)
+    - [Run Skycoin with options](#run-skycoin-with-options)
+- [API Documentation](#api-documentation)
+    - [Wallet REST API](#wallet-rest-api)
+    - [JSON-RPC 2.0 API](#json-rpc-20-api)
+    - [Skycoin command line interface](#skycoin-command-line-interface)
+- [Development](#development)
+    - [Modules](#modules)
+    - [Formatting](#formatting)
+    - [Running tests](#running-tests)
+    - [Dependency Management](#dependency-management)
+    - [Wallet GUI Development](#wallet-gui-development)
+    - [Releases](#releases)
+
+<!-- /MarkdownTOC -->
+
 ## Installation
 
-### Installing golang
+### Installing Go
+
+[Install go1.9+](https://golang.org/doc/install)
+
+### Set GOPATH in your environment
+
+Create a folder for your `$GOPATH`, for example:
+
+```sh
+mkdir ~/.gopath
```

Comment:
why use hiden folder: ~/.gopath, the default GOPATH is $HOME/go since go1.8.

Reply:
oh didn't know they default now

Reply:
you don't need to set gopath anymore?